### DISCUSSION TITLE
xcode beta 15 needed?

### DISCUSSION THREAD

**Discussion by Mrblutziii** [Whisky-App/Whisky#291] [Q&A] [answered]
Hey guys, I am not a programmer. Just a gamer trying to get some games onto my mac. I went through the wiki and accomplished everything that was asked. Got both Battle.net and Steam running and installed Overwatch, as well as attempted battlebit. Both games would start and then stop (Overwatch would show a window saying it crashed and Battlebit would just say it was running on steam and then go back to the "play now" button).

From the previous guides I looked at, it looks like they ask you to install the newest Xcode beta? Is that a requirement when using whiskey? 

I have attempted using the Beta 3 and the first Beta for the Game Porting toolkit, and neither allowed me to play the games. Are these games just currently not playable, or am I missing something? Thanks so much.

**Comment by IsaacMarovitz** [comment, marked as answer]
You do not need to install Xcode to use Whisky.
